h2o-docs/src/product/data-science/drf.rst

```diff
@@ -220,6 +222,8 @@ Defining a DRF Model
 	- QuantilesGlobal
 	- RoundRobin
 
+  **Note**: H2O supports extremely randomized trees (XRT) via ``histogram_type="Random"``. When this is specified, the algorithm will sample N-1 points from min...max and use the sorted list of those to find the best split. The cut points are random rather than uniform. For example, to generate 4 bins for some feature ranging from 0-100, 3 random numbers would be generated in this range (13.2, 89.12, 45.0). The sorted list of these random numbers forms the histogram bin boundaries e.g. (0-13.2, 13.2-45.0, 45.0-89.12, 89.12-100).
```

Comment:
I think moving this to a separate section called "Extremely Randomized Trees" (right before "Interpreting a DRF model") would be helpful.  Here is another piece of text that we can borrow from sklearn which could help explain: 

"In extremely randomized trees (XRT), randomness goes one step further in the way splits are computed. As in random forests, a random subset of candidate features is used, but instead of looking for the most discriminative thresholds, thresholds are drawn at random for each candidate feature and the best of these randomly-generated thresholds is picked as the splitting rule. This usually allows to reduce the variance of the model a bit more, at the expense of a slightly greater increase in bias."

We could lead the section with this paragraph, and then add the "Note" paragraph that you have above.